flterm.c

```diff
@@ -36,6 +36,7 @@
 #include <unistd.h>
```

Comment:
You seem to have lots of extra newlines in a couple of places?

Reply:
Can you also add doxygen style comments to your new functions?

Reply:
I feel like the `--help` needs some type of update too?

Reply:
Two things I can think of saying:
"If a kernel_image is passed, flterm will listen for either an XMODEM start number (if `--xmodem` is passed) or SFL magic number, whichever comes first."

(I haven't implemented the `--xmodem` option yet)

"To avoid spurious transfers due to the XMODEM start character being within the printable ASCII set, an XMODEM session is only attempted once after invocation."